public/apps/configuration/panels/tenant-list/save_changes_modal.tsx

```diff
@@ -0,0 +1,180 @@
+      <EuiConfirmModal
+        // onClose={props.handleClose}
+        style={{ width: 600 }}
+        title="Change global default tenant?"
+        onCancel={props.handleClose}
+        onConfirm={async () => {
+          await props.handleSave(props.updatedTenancyConfig);
+        }}
+        cancelButtonText="Discard Changes"
+        confirmButtonText="Change Default Tenant"
+        defaultFocusedButton="confirm"
+      >
+        <p>
+          Users will load into {props.updatedTenancyConfig.default_tenant} tenant when they log into Dashboards
+          if they have the appropriate permissions. If users don’t have permissions to a custom
+          tenant they will load into the global tenant.
+          <ExternalLink href={DocLinks.TenantPermissionsDoc} />
+        </p>
+      </EuiConfirmModal>
+    );
+    return globalDefaultModal;
+  }
+
+  const [tenancyChecked, setTenancyChecked] = React.useState(false)
+  const [priavteTenancyChecked, setPrivateTenancyChecked] = React.useState(false)
+  const [defaultTenantChecked, setDefaultTenantChecked] = React.useState(false)
+
+  let tenancyToggled = false;
+  let privateTenantToggled = false;
+  let defaultTenantChanged = false;
+  let tenancyChangeCheckbox;
+  let privateTenancyChangeCheckbox;
+  let defaultTenantChangeCheckbox;
+  let tenancyChangeMessage = "Message 1";
+  let privateTenancyChangeMessage = "Message 2";
+  let defaultTenantChangeMessage = "Message 3";
+
+  if(props.updatedTenancyConfig.multitenancy_enabled!=props.originalTenancyConfig.multitenancy_enabled){
+    tenancyToggled = true;
+    if(props.updatedTenancyConfig.multitenancy_enabled) {
+      tenancyChangeMessage="Enable Tenancy - Users will be able to make use of the Tenancy Feature " +
+        "in OpenSearch Dashboards and switch between tenants they have access to."
+    }
+    else{
+      tenancyChangeMessage="Disabling Tenancy - Users won't be able to access index patterns, visualizations, " +
+        "dashboards, and other OpenSearch Dashboards objects saved in tenants other than the " +
+        "global tenant."
+    }
+    tenancyChangeCheckbox=(
+      <EuiCheckbox
+        id={"tenancyChangeCheckbox"}
+        label={tenancyChangeMessage}
+        checked={tenancyChecked}
+        onChange={() => setTenancyChecked(!tenancyChecked)}
+      >
+      </EuiCheckbox>
+    )
+
+  }
+
+  if((props.updatedTenancyConfig.private_tenant_enabled!=props.originalTenancyConfig.private_tenant_enabled) && props.updatedTenancyConfig.multitenancy_enabled){
+    privateTenantToggled = true;
+    if(props.updatedTenancyConfig.private_tenant_enabled) {
+      privateTenancyChangeMessage="Enable private tenant - Users will be able to create index patterns, visualizations, and" +
+        "other OpenSearch Dashboards objects in a private tenant that they only have access to."
+    }
+    else{
+      privateTenancyChangeMessage="Disabling private tenant - Users won't be able to access index patterns, visualizations, and" +
```

Comment:
Maybe I am reading this wrong, but would this say the "Disabling..." message even if `updatedTenancy` was `private_tenant_enabled` and `origingTenancy` was `private_tenant_enabled`? Is it possible to reach here without the config update being different? I.e. someone sets the config to what it already is?